tests/util/transactions.ts

```diff
@@ -227,3 +229,10 @@ export async function callPrecompile(
     },
   ]);
 }
+
+/// Sign and sand Substrate transaction then create a block.
```

Comment:
Not sure it will sand much :)
Also not sure a function is needed for call 1 single function :/

Reply:
>Not sure it will sand much :)

haha I will correct it

>Also not sure a function is needed for call 1 single function :/

It is not very useful indeed but the idea is that it's easier to write `const events2 = blabla` than `const {events:events2}= blabla` 
Should I go ahead and remove that util?

Reply:
You can keep it